.github/workflows/build-gtk.yml

```diff
@@ -0,0 +1,69 @@
+      DotnetVersion: 8.0.200
+      PROJECTS_TO_PACK: |
+        src/BlazorWebView/src/Maui/Microsoft.AspNetCore.Components.WebView.Maui.csproj
+        src/Essentials/src/Essentials.csproj
+        src/Controls/src/Core/Controls.Core.csproj
+        src/Controls/src/Xaml/Controls.Xaml.csproj
+        src/Core/src/Core.csproj
+        src/Compatibility/Core/src/Compatibility.csproj
+        src/Controls/Foldable/src/Controls.Foldable.csproj
+    steps:
+      - name: Checkout MAUI repo
+        uses: actions/checkout@v2
+      - name: Setup .NET SDK ${{ env.DotnetVersion }}
+        uses: actions/setup-dotnet@v1.7.2
+        with:
+          dotnet-version: ${{ env.DotnetVersion }}
+      - name: Install gtk workload
+        run: |
+            # For some reason automatic workload manifest detection doesn't work (see https://github.com/GtkSharp/GtkSharp/issues/355#issuecomment-1446262239), so download and uzip mainfest file manually
+            dotnet nuget add source --name nuget.org "https://api.nuget.org/v3/index.json"
+            wget https://www.nuget.org/api/v2/package/gtksharp.net.sdk.gtk.manifest-${{ env.GtkSharpManifestVersion }}/$GtkSharpVersion -O gtksharp.net.sdk.gtk.manifest-${{ env.GtkSharpManifestVersion }}.nupkg            
+            DOTNET_DIR=/home/runner/.dotnet
+            WORKLOAD_MANIFEST_DIR=$DOTNET_DIR/sdk-manifests/${{ env.DotnetVersion }}/gtksharp.net.sdk.gtk
+            mkdir -p $WORKLOAD_MANIFEST_DIR
+            unzip -j gtksharp.net.sdk.gtk.manifest-${{ env.GtkSharpManifestVersion }}.nupkg "data/*" -d $WORKLOAD_MANIFEST_DIR/
+            rm gtksharp.net.sdk.gtk.manifest-${{ env.GtkSharpManifestVersion }}.nupkg
+            chmod 764 $WORKLOAD_MANIFEST_DIR/*
+            dotnet workload search
+            dotnet workload install gtk --skip-manifest-update
+      - name: Build MAUI
+        run: |
+            mv Directory.Build.Override.props.in Directory.Build.Override.props
+            dotnet build Microsoft.Maui.BuildTasks.slnf
+            echo "$PROJECTS_TO_PACK" | while read -r project; do
+              [ -z "$project" ] && continue  # Skip if empty
+              dotnet build -c Release "$project"
+            done
+      - name: Pack MAUI
+        run: ./pack_mali.sh
+      - name: Upload binaries to nuget (if tag or main branch, and nugetKey is present)
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NUGET_KEY: ${{ secrets.NUGET_KEY }}
+        if: ${{ env.NUGET_KEY != ''  && (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/')) }}
+        run: dotnet nuget push **/*.nupkg --api-key ${{ secrets.NUGET_KEY }} --source https://api.nuget.org/v3/index.json --skip-duplicate
+
+  dotnet-format:
+    needs: build_and_test
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Run dotnet format
+        run: |
+          # remark: doesnt work, alwasy fails:
```

Comment:
@webwarrior-ws typo ^ but wait, why did you add this dotnet format thing? looks unrelated

Reply:
I copied the whole `build-gtk.yml` file

Reply:
from where??

Reply:
from `maui-linux`

Reply:
did you put this in commit msg? still, better to copy only what is relevant, we will pull from maui-linux last version when we need to and then we get all those things

Reply:
Removed `dotnet-format` job